include/mme-app/mmeStates/attachStates.h

```diff
@@ -219,6 +219,46 @@
 			AttachWfIdentityResponse();  
 	};
 	
+	class AttachWfIdentificationResponse : public AttachState
```

Comment:
Can you please present call flow which you are implementing ? It will help in understanding the code.

Reply:
We have replaced the identity request to UE with identification request to old MME. So for now the implemented call flow doesn't include identity request at all.
Moving further, we will be adding the same while considering the negative scenarios.